subxt/src/book/mod.rs

```diff
@@ -0,0 +1,134 @@
+//!   one node will often "Just Work" when pointed at different nodes that use the same pallet.
+//! - Subxt can work offline; you can generate and sign transactions, access constants from node
+//!   metadata and more, without a network connection. This is all checked at compile time, so you
+//!   can be certain it won't try to establish a network connection if you don't want it to.
+//! - Subxt can forego the statically generated interface and build transactions, storage queries
+//!   and constant queries using data provided at runtime, rather than queries constructed
+//!   statically.
+//! - Subxt can be compiled to WASM to run in the browser, allowing it to back Rust based browser
+//!   apps, or even bind to JS apps.
+//!
+//! ## Limitations
+//!
+//! In various places, you can provide a block hash to access data at a particular block, for
+//! instance:
+//!
+//! - [`crate::storage::StorageClient::at`]
+//! - [`crate::events::EventsClient::at`]
+//! - [`crate::blocks::BlocksClient::at`]
+//! - [`crate::runtime_api::RuntimeApiClient::at`]
+//!
+//! However, Subxt is (by default) only capable of properly working with blocks that were produced
+//! after the most recent runtime update. This is because it uses the most recent metadata given
+//! back by a node to encode and decode things. It's possible to decode older blocks produced by a
+//! runtime that emits compatible (currently, V14) metadata by manually setting the metadata used by
+//! the client using [`crate::client::OnlineClient::set_metadata()`].
+//!
+//! Subxt does not support working with blocks produced prior to the runtime update that introduces
+//! V14 metadata. It may have some success decoding older blocks using newer metadata, but may also
+//! completely fail to do so.
+//!
+//! ## Quick start
+//!
+//! Here is a simple but complete example of using Subxt to transfer some tokens from Alice to Bob:
+//!
+//!
+//! ```rust,ignore
+#![doc = include_str!("../../../examples/examples/balance_transfer_basic.rs")]
+//! ```
+//!
+//! This example assumes that a Polkadot node is running locally (Subxt endeavors to support all
+//! recent releases). Typically, to use Subxt to talk to some custom Substrate node (for example a
+//! parachain node), you'll want to:
+//!
+//! 1. [Generate an interface](setup::codegen).
+//! 2. [Configure and instantiate the client](setup::client).
+//!
+//! Follow the above links to learn more about each step.
+//!
+//! ### Acquiring metadata
+//!
+//! The simplest way to do acquire metadata is to use the `subxt` CLI tool to download it from your
+//! node. The tool can be installed via `cargo`:
+//!
+//! ```shell
+//! cargo install subxt-cli
+//! ```
+//!
+//! Once installed, run `subxt metadata > metadata.scale` to download the node's metadata and save
+//! it to some file (again, this assumes that the node you'd like to talk to is running locally; run
```

Comment:
```suggestion
//! it to a file, in this case`metadata.scale` (again, this assumes that the node you'd like to talk to is running locally; run
```

Reply:
Removed this section because I'd moved it to a separate page linked above :)